core/src/main/scala/org/scalarelational/instruction/InsertMultiple.scala

```diff
@@ -1,18 +1,17 @@
 package org.scalarelational.instruction
 
+import org.scalarelational.table.Table
 import org.scalarelational.column.ColumnValue
 
 /**
  * @author Matt Hicks <matt@outr.com>
  */
-case class InsertMultiple(rows: Seq[Seq[ColumnValue[_]]]) extends Insert with Instruction[List[Int]] {
-  override def table = rows.head.head.column.table
+case class InsertMultiple(table: Table, rows: Seq[Seq[ColumnValue[_]]])
+  extends Insert[List[Int]] with Instruction[List[Int]] {
```

Comment:
This would be only necessary if we wanted to get ``Ref``s for insertions of multiple rows. Currently, there is no way to insert multiple ``MappedTable`` objects at the same time. This is something we may discuss, but it's not in the scope of this pull request.